packages/manager/.editorconfig

```diff
@@ -0,0 +1,15 @@
+# editorconfig.org
+
+[*]
+indent_style = tab
```

Comment:
Yes, we use tabs rather than spaces in all of the new packages. I recommend we convert `slice-machine-ui` to tabs as well, but we didn't want to alter existing code too much.

You can see @bapmrl comment re: tabs and my reply here: https://www.notion.so/prismic/Slice-Machine-Review-Repository-Infrastructure-2023-02-08-Baptiste-1e4882f5065044509b6d2835b1b6dd07?pvs=4#c2b79a6855fa4601807aaed357d35fe6